docs/5-external-interface/README.md

```diff
@@ -1,12 +1,12 @@
 # External interface
 
+// Note: is there any point in this file if it's just duplicating the entry points?
```

Comment:
Didn't find this file useful, since the entry points seem to already cover this.

Perhaps we should either dedupe, or discuss which events are emitted instead?